test_images/jpeg2000/README.md

```diff
@@ -0,0 +1,24 @@
+from https://github.com/uclouvain/openjpeg-data
+
+```
+These files were originally developed by Algo Vision Technology GmbH,
+Aware Inc., Kodak Inc., and Ricoh Innovations Inc., in the course of
+development of ITU-T 803 | ISO/IEC 15444-4.  The files help test parts
+of implementations of a part of ITU-T 800 | ISO/IEC 15444-1. Copyright
+holders agree not to assert against ITU, ISO/IEC and users of the JPEG
+2000 Standards (Users) any of their rights under the copyright, not
+including other intellectual property rights, for these files with
+respect to the usage by ITU, ISO/IEC and Users of these files or
+modifications thereof for use in hardware or software products
+claiming conformance to or testing conformance to the JPEG 2000
+Standard.  The original developers of these files, ITU and ISO/IEC
+assume no liability for use of these files or modifications
+thereof. No right to these files is granted for non JPEG 2000 Standard
+uses. Copyright holders have full right to use these files for his/her
+own purpose, assign or donate these files to any third party and to
+inhibit third parties from using this software module for non JPEG
+2000 Standard conforming products. This copyright notice must be
+included in all copies or derivative works of these files.
+
+Copyright (c) 2002.
+```
```

Comment:
Quote instead of code formatting?

```suggestion
> These files were originally developed by Algo Vision Technology GmbH,
Aware Inc., Kodak Inc., and Ricoh Innovations Inc., in the course of
development of ITU-T 803 | ISO/IEC 15444-4.  The files help test parts
of implementations of a part of ITU-T 800 | ISO/IEC 15444-1. Copyright
holders agree not to assert against ITU, ISO/IEC and users of the JPEG
2000 Standards (Users) any of their rights under the copyright, not
including other intellectual property rights, for these files with
respect to the usage by ITU, ISO/IEC and Users of these files or
modifications thereof for use in hardware or software products
claiming conformance to or testing conformance to the JPEG 2000
Standard.  The original developers of these files, ITU and ISO/IEC
assume no liability for use of these files or modifications
thereof. No right to these files is granted for non JPEG 2000 Standard
uses. Copyright holders have full right to use these files for his/her
own purpose, assign or donate these files to any third party and to
inhibit third parties from using this software module for non JPEG
2000 Standard conforming products. This copyright notice must be
included in all copies or derivative works of these files.
>
> Copyright (c) 2002.
```

Reply:
I used code formatting so that spaces/newlines would be preserved exactly, but it probably doesn't matter.